src/client/api/pool.c

```diff
@@ -14,36 +14,6 @@
 /** Disable backward compat code */
 #undef daos_pool_connect
 
-/** Kept for backward ABI compatibility, but not advertised via header file */
-int
-daos_pool_connect(const char *pool, const char *grp, unsigned int flags,
-		  daos_handle_t *poh, daos_pool_info_t *info, daos_event_t *ev)
-{
-	daos_pool_connect_t	*args;
-	tse_task_t		*task;
-	const unsigned char	*uuid = (const unsigned char *) pool;
-	int			rc;
-
-	DAOS_API_ARG_ASSERT(*args, POOL_CONNECT);
-	if (!daos_uuid_valid(uuid))
-		return -DER_INVAL;
-
-	rc = dc_task_create(dc_pool_connect, NULL, ev, &task);
-	if (rc)
-		return rc;
-
-	args = dc_task_get_args(task);
-	args->grp	= grp;
-	args->flags	= flags;
-	args->poh	= poh;
-	args->info	= info;
-	/** use deprecated field for backward compatibility */
-	uuid_copy((unsigned char *)args->uuid, uuid);
```

Comment:
args->uuid can also likely be removed from daos_pool_connect_t, but this can be cleaned up in a separate patch.

Reply:
ok. I can remove "uuid_t	uuid" from daos_pool_connect_t in a separate patch.